config/go-app-ussd_higherhealth_healthcheck.afr_ZA.json

```diff
@@ -0,0 +1,262 @@
+        null,
+        "Jy kan enige tyd na hierdie diens terug keer.\nOnthou, as jy dink dat jy COVID-19 het, BLY BY DIE HUIS, vermy kontak met ander mense en self-kwarantyn."
+    ],
+    "It's not a substitute for professional medical advice/diagnosis/treatment. Get a qualified health provider's advice about your medical condition/care.": [
+        null,
+        "Die is nie ’n plaasvervanger vir professionele advies/ diagnose/ behandeling nie. Kry ’n gekwalifiseerde dokter se advies oor jou mediese sorg."
+    ],
+    "Next": [
+        null,
+        "Volgende"
+    ],
+    "You confirm that you shouldn't disregard/delay seeking medical advice about treatment/care because of this service. Rely on info at your own risk.": [
+        null,
+        "Jy bevestig dat jy nie agv die diens enige mediese advies oor sorg sal ignoreer/verontagsaam nie. Maak staat op inligting op jou eie risiko."
+    ],
+    "Please TYPE your first name": [
+        null,
+        ""
+    ],
+    "Please TYPE your surname": [
+        null,
+        ""
+    ],
+    "Select your province\n\nReply:": [
+        null,
+        "Kies jou Provinsie\n\nAntwoord:"
+    ],
+    "EASTERN CAPE": [
+        null,
+        "OOS-KAAP"
+    ],
+    "FREE STATE": [
+        null,
+        "VRYSTAAT"
+    ],
+    "GAUTENG": [
+        null,
+        "GAUTENG"
+    ],
+    "KWAZULU NATAL": [
+        null,
+        "KWAZULU-NATAL"
+    ],
+    "LIMPOPO": [
+        null,
+        "LIMPOPO"
+    ],
+    "MPUMALANGA": [
+        null,
+        "MPUMALANGA"
+    ],
+    "NORTH WEST": [
+        null,
+        "NOORDWES"
+    ],
+    "NORTHERN CAPE": [
+        null,
+        "NOORD-KAAP"
+    ],
+    "WESTERN CAPE": [
+        null,
+        "WES-KAAP"
+    ],
+    "Please TYPE the name of your Suburb, Township, Town or Village (or nearest)": [
+        null,
+        "TIK asb die naam in van jou Voorstad, Township, Dorp of Village (of naaste)"
+    ],
+    "How old are you?": [
+        null,
+        "Hoe oud is jy?"
+    ],
+    "How old are you?\n": [
+        null,
+        "Hoe oud is jy?\n"
+    ],
+    "<18": [
+        null,
+        ""
+    ],
+    "18-39": [
+        null,
+        ""
+    ],
+    "40-65": [
+        null,
+        ""
+    ],
+    ">65": [
+        null,
+        ""
+    ],
+    "Do you feel very hot or cold? Are you sweating or shivering? When you touch your forehead, does it feel hot?\n\nReply": [
+        null,
+        "Was jy al gediagnoseer as Oorgewig of met Diabetes, Hoë Bloeddruk of ’n Hart kondisie?\n\nAntwoord"
+    ],
+    "Please use numbers from list. Do you feel very hot or cold? Are you sweating or shivering? When you touch your forehead, does it feel hot?\n\nReply": [
+        null,
+        "Voel jy baie warm of koud? Sweet of bewe jy? Wanneer jy aan jou voorkop vat, voel dit warm?\n\nAntwoord"
```

Comment:
We do have translations for the `catchall` error, so for the catchall error it should be simple enough to concatenate the error translation with the question translation.

We'll just need to be careful around character counts.